content/lessons/transacting/transacting-1/index.tsx

```diff
@@ -1,41 +1,36 @@
 'use client'
 
-import { Button, Text, Title } from 'ui'
+import { Intro } from 'ui'
 
 export default function Transacting() {
-  return (
-    <div className="grid w-full grid-cols-1 justify-center justify-items-center">
-      <div className="flex w-full items-center px-6 text-white lg:w-1/2 lg:px-0">
-        <div className="content-center justify-items-center px-1 font-nunito">
-          <div className="genesis">
-            <Title>{`What's in a transaction?`}</Title>
-
-            <Text className="pt-2 text-lg">
-              Bitcoin is censorship resistant money. Anybody can send money by
-              broadcasting a transaction to the network. After broadcast,
-              transactions are packaged up into blocks by miners. Miners compete
-              against other miners for the privilege of building on the chain.
-            </Text>
-
-            <Text className="pt-2 text-lg">
-              Transactions have two parts: inputs and outputs. In the previous
-              exercise we decoded a secret message found inside a transaction
-              input. This time we’ll decode a message that belongs to the output
-              part.
-            </Text>
+  const text = [
```

Comment:
Hmm I'm not sure I like this approach, I think I would prefer just using children in this case. I noticed the conversation you guys had about flexibility and I think it's perfectly fine to use `children` instead of this json approach. So:

```jsx
<Intro
  title={}
  next={}
>
  <p class="pt-2 text-lg">...</p>
  <p class="pt-2 text-lg">...</p>
  <p class="pt-2 text-lg">...</p>
</Intro>
```

Then in Intro.tsx:
```jsx
<div className="genesis">
  <Title>{title}</Title>
  {children}
</div>
```

That way I can deviate from simple `<p>` tags with some css class more easily as a contributor.